components/perf-tools/geopm/SPECS/geopm.spec

```diff
@@ -0,0 +1,570 @@
+#  Copyright (c) 2015, 2016, 2017, 2018, Intel Corporation
+#
+#  Redistribution and use in source and binary forms, with or without
+#  modification, are permitted provided that the following conditions
+#  are met:
+#
+#      * Redistributions of source code must retain the above copyright
+#        notice, this list of conditions and the following disclaimer.
+#
+#      * Redistributions in binary form must reproduce the above copyright
+#        notice, this list of conditions and the following disclaimer in
+#        the documentation and/or other materials provided with the
+#        distribution.
+#
+#      * Neither the name of Intel Corporation nor the names of its
+#        contributors may be used to endorse or promote products derived
+#        from this software without specific prior written permission.
+#
+#  THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+#  "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+#  LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+#  A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+#  OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+#  SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+#  LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+#  DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+#  THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+#  (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY LOG OF THE USE
+#  OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+#
```

Comment:
@koomie I am curious about license text in our SPEC files. Isn't OpenHPC content supposed to be Apache 2.0 licensed? Does it make sense to have a different license text in SPEC files? I understand that the actual license of the source code is different, that makes sense. But I think the SPEC file should be Apache 2.0 and without an additional license text in the SPEC file.

Reply:
Chris and I don't have approval (OSPDT) to release GEOPM under any other license than the BSD-3-clause, so I'm not sure we're empowered to change this if it is deemed necessary.

Reply:
I agree, this is a recapitulation of the license for the source release itself which is flagged using normal RPM semantics via the `License:       BSD-3-Clause` line and inclusion of formal `LICENSE` file which ships with the package.  We are not 100% consistent, but the vast majority of the .specs in ohpc do not do this.  Would you be amenable to contributing a spec with the standard ohpc .spec header?

Reply:
Thanks for the feedback.  I'm going to remove the license header that's there now and replace it with just the standard ohpc header.  I'll have that up in a few minutes.